WooCommerce/Classes/ViewRelated/Orders/OrderListViewModel.swift

```diff
@@ -250,8 +250,23 @@ final class OrderListViewModel {
             return NSCompoundPredicate(andPredicateWithSubpredicates: predicates)
         }()
 
+        let predicateSalesChannel: NSPredicate? = {
+            guard let salesChannelFilter = filters?.salesChannel else {
+                return nil
+            }
+
+            switch salesChannelFilter {
+            case .pointOfSale:
+                let predicate = NSPredicate(format: "createdVia == %@", "pos-rest-api")
+                return predicate
+            case .any:
+                return nil
```

Comment:
Using it, I immediately felt that there should be a `web` option. 

Other options I've seen in the backend are `checkout` and `rest-api` – presumably there are more. `checkout` came from both the block and classic checkouts, but I did see `store_api` for a draft order – it got changed to `checkout` when the order was paid for though.

I don't know how much you've discussed it... even though the fix is out of scope of this PR, it _feels_ wrong from a user perspective right now. Selecting "not POS" would be fine but isn't possible right now, as far as I can see.

Reply:
I don't think it was discussed at the time, but agree: Currently wp-admin already shows a sales channel filter within orders for `All Sales Channels`, `Admin`, `Checkout`, and `Point of Sale`. So we should do the same in-app

<img width="685" height="164" alt="Screenshot 2025-07-14 at 12 17 59" src="https://github.com/user-attachments/assets/5d3377df-d306-4e00-be6c-de6b0544d2ef" />